src/ranking/effectiveness/mark.ts

```diff
@@ -262,8 +262,8 @@ function init() {
     DISCRETE_OR_NONE.forEach((yType) => {
       // has occlusion
       const ddMark = {
-        point: 0,
         rect: 0,
+        point: -0.0025,
```

Comment:
I wonder if there are corner cases where making point inferior to rect in this case cause a bit more problem.

Reply:
Since rect should only compete with point in a DxDx(Q|O) query, there should only be two cases right? (seen below).

![screen shot 2017-11-19 at 4 07 09 pm](https://user-images.githubusercontent.com/11954298/32997886-77a0316a-cd4a-11e7-8bf0-e7b45aa3f4c6.png)
![screen shot 2017-11-19 at 4 54 31 pm](https://user-images.githubusercontent.com/11954298/32997887-77bbe996-cd4a-11e7-9f6d-e1d59b2f04d2.png)

Reply:
Ah another case where we specific color and therefore Rect should beat Point

![image](https://user-images.githubusercontent.com/11954298/32998137-679ed886-cd4d-11e7-8a1c-99ac864243be.png)

Reply:
If there is x:D,y:D,?:Q where Q is not aggregated, then perhaps points are better because at least users can notice the occlusion. 

If there is ?:D,?:D,?:Q  where Q is not aggregated, perhaps, having one of the Q on x or y maybe better?

We should also consider the cases for Q is aggregated -- that case is actually less problematic.

Reply:
For now assume occlusion heuristics are correct and modify rankings accordingly

Reply:
DxD Point + size >  DxD Rect + color > DxD Point + color

- Add a feature color channel to correct this.

- Because point > rect and point probably get 0; rect will get small minus (-R).  the rect x color can be a +R  and maybe do something small with point and color (minus  < |R|)